src/dmd/cli.d

```diff
@@ -256,13 +256,16 @@ dmd -cov -unittest myprog.d
             (only imports).`,
         ),
         Option("dip25",
-            "implement http://wiki.dlang.org/DIP25"
+            "implement https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md",
+            "implement $(LINK2 https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md, DIP25 (Sealed references))"
```

Comment:
Should be a capital R for "references" to make it consistently cased with the others